_posts/2019-01-03-label-based-prs.markdown

```diff
@@ -0,0 +1,197 @@
+---
+layout: epic
+title: Migrating to PR Label based deployments
+date: 2019-01-03
+author: [orta]
+categories: [build tools, deployment]
```

Comment:
road and bridges!

Reply:
Should make a series on this, might be enough already